### PR TITLE
Etienne improve highlight

### PR DESCRIPTION
**Pull request by RichartE**
Decoupled graph from highlighting and improved marking algorithm

### CODE REVIEW

File: wikichange/src/content/content.js [hunk 1]
```diff
@@ -344,8 +344,11 @@ getPageCreationDate(title).then((date) => {
     promises.push(renderGraphOverlay());
     promises.push(renderItemsBelowGraph(date));
 
-    // Render popups only when graph and buttons are loaded
-    Promise.all(promises).then(() => renderPopup());
+    // Render popups and initial highlight only when graph and buttons are loaded
+    Promise.all(promises).then(([, [curRevisionId, oldRevisionId]]) => {
+        renderPopup();
+        highlightRevisionBetweenRevisionIds(title, curRevisionId, oldRevisionId)
```
Comment (sukritsangvong) [inline wikichange/src/content/content.js:350]: nice

**Review by sukritsangvong — APPROVED** [review]
Looks good

**Comment by sukritsangvong** [comment]
Can merge after the merge conflicts are resolved

**Comment by LuisaE** [comment]
@etiennerichart can you merge master in this branch?

**Pull request merged**